server/db.js

```diff
@@ -9,7 +9,6 @@ var pool = mysql.createPool({
   password: process.env.MYSQL_PASSWORD || '',
   database: process.env.MYSQL_DATABASE || 'rnn_generator',
   port: process.env.MYSQL_PORT || 3306,
-  ssl: "Amazon RDS",
```

Comment:
This is required for AWS, buy removing it, the app will not be able to connect to RDS